prottp.go

```diff
@@ -37,10 +40,22 @@ func Wrap(service Service) http.Handler {
 	return mux
 }
 
+var marshaler = jsonpb.Marshaler{
+	EnumsAsInts:  false,
+	EmitDefaults: false,
```

Comment:
Suggest `true`?

Can the `.proto` file have default values in it? Ah, no: https://developers.google.com/protocol-buffers/docs/proto3#default

So if you look at the JSON output, you won't see *every* field...